mkdocs.yml

```diff
@@ -113,6 +113,7 @@ nav:
     - 'Install from source': ftldns/compile.md
     - 'dnsmasq warnings': ftldns/dnsmasq_warn.md
     - 'Cache dump': ftldns/cache_dump.md
+    - 'internal pcap packet dump': ftldns/package_dump.md
```

Comment:
This is a very long name and you are putting it into a group that already hay way too items

![Screenshot from 2022-01-14 22-22-15](https://user-images.githubusercontent.com/16748619/149586984-45297731-80eb-4d04-9eb8-f2a34cfb86de.png)

We should think about some restructuring here. It's not something the average user will need. It can go under some advanced/insiders/whatever category. It doesn't even have to be under `FTLDNS` necessarily

Reply:
I restructured the menu

Main:
![Bildschirmfoto zu 2022-01-30 21-25-51](https://user-images.githubusercontent.com/26622301/151716447-9109ff14-41d3-4cd8-bf29-61f198fbd908.png)

Advanced:
![Bildschirmfoto zu 2022-01-30 21-25-55](https://user-images.githubusercontent.com/26622301/151716448-01043905-9956-4ead-ba11-9ee6761fc583.png)

Debugging
![Bildschirmfoto zu 2022-01-30 21-25-59](https://user-images.githubusercontent.com/26622301/151716449-60bc89e2-02c7-4dfc-bb4d-0cd503a25a5c.png)